### DISCUSSION TITLE
Label style issue

### DISCUSSION THREAD

**Discussion by woolson** [community/community#4160] [Mobile] [answered]
## General information

Platform: iOS@15.0
Github Mobile Ver: 1.17.0

## Screenshot 

![image](https://user-images.githubusercontent.com/27878293/121754298-d5337280-cb46-11eb-995e-edb7b49b54ec.jpeg)

**Comment by alcere** [comment, marked as answer]
Hey thanks for your feedback. I've created an issue in our internal repo to fix this. Please keep the feedback coming!
